exercises/oop/geometric_shapes/deivio-solution/Square.js

```diff
@@ -0,0 +1,23 @@
+export default
+
+class Square{
+    constructor(Side) {
+        this._side = Side;
+    }
+
+    get side() {
+        return this._side;
+    }
+
+    set side(Side) {
+        this._side = Side;
+    }
+
+    calculateArea() {
+        return this._side *this._side;
```

Comment:
If we have getters, why not use them?
this._side accesses the field directly. We want encapsulation